code/config.h

```diff
@@ -393,7 +393,7 @@
 
 
 /* Pool MVT Configuration -- see <code/poolmv2.c> */
-/* FIXME: These numbers were lifted from mv2test and need thought. */
+/* TODO: These numbers were lifted from mv2test and need thought. */
```

Comment:
This comment may need more explanation about the impact of the number choices.

Reply:
Raise: #176